src/orders/components/OrderCustomerAddressesEditDialog/OrderCustomerAddressesEditDialog.tsx

```diff
@@ -36,8 +36,14 @@ import OrderCustomerAddressesEditForm, {
 } from "./form";
 import { dialogMessages } from "./messages";
 import OrderCustomerAddressEdit from "./OrderCustomerAddressEdit";
+import OrderCustomerAddressesSearch from "./OrderCustomerAddressesSearch";
 import { useStyles } from "./styles";
+import { validateDefaultAddress } from "./utils";
 
+export interface OrderCustomerSearchAddressState {
```

Comment:
Connected to the comments below I think this type should be a bit different. Notice that while having the `type` optional, ts won't throw an error when we accidentally set `{ open: true }` without a type. Therefore, what we actually want here is always have the type present, with default data I described in the comment below

Reply:
Will do